src/test/resources/features/fixtures/csvw/test001.csv

```diff
@@ -0,0 +1,9 @@
+Surname,FamilyName
```

Comment:
And make sure all of these things which are auto-downloaded are removed too!

Reply:
done